package.json

```diff
@@ -24,7 +24,7 @@
     "lint:js": "run-s prettier:js eslint",
     "lint:sass": "stylelint \"**/*.scss\" --fix",
     "eleventy": "eleventy",
-    "eleventy:serve": "eleventy --serve --incremental",
+    "eleventy:serve": "eleventy --serve",
```

Comment:
@mirisuzanne Did this fix your server errors?

Reply:
it did, yea